packages/grafana-data/src/themes/createShape.ts

```diff
@@ -19,10 +40,13 @@ export interface ThemeShapeInput {
 }
 
 export function createShape(options: ThemeShapeInput): ThemeShape {
-  const baseBorderRadius = options.borderRadius ?? 2;
+  const baseBorderRadius = options.borderRadius ?? 6;
 
   const radius = {
     default: `${baseBorderRadius}px`,
+    md: `${baseBorderRadius}px`,
+    sm: `${Math.ceil(baseBorderRadius * (2 / 3))}px`, // for default base becomes 4
+    lg: `${Math.ceil(baseBorderRadius * (5 / 3))}px`, // for default base becomes 10
```

Comment:
maybe we should also extend the `ThemeShapeInput` to also allow for passing `borderRadiusSm`/`borderRadiusLg`? 🤔 then we could define them similarly to line 43, e.g. 
```tsx
const borderRadiusSm = options.borderRadiusSm ?? 4;
const borderRadiusLg = options.borderRadiusLg ?? 10;
```
and it gives themes more flexibility to override